agents/triage_agent.py

```diff
@@ -253,7 +253,22 @@ def render_prompt(input: InputSchema) -> str:
            is clarification-needed
          * This is the correct choice when you are sure a problem exists but cannot find the solution yourself
 
-         2.5 Set the Jira fields as per the instructions below.
+         2.5. If the chosen action is backport then search for the issue reproducer in the issue comments.
```

Comment:
I suggest using `reproducer test case` instead of `reproducer`, everywhere, not just here. Reproducer test case succeeds when the reproducer fails - the issue didn't manifest.